integration-test/662-basic-outdoor-pois.py

```diff
@@ -3,10 +3,11 @@
     16, 10550, 25297, 'pois',
     { 'kind': 'bbq', 'min_zoom': 18 })
 
-#http://www.openstreetmap.org/node/3497698404
+# Node: Valencia Cyclery (3443701422)
+# http://www.openstreetmap.org/node/3443701422
```

Comment:
Huh, someone tagged the repair counter of that bike shop as `amenity= bicycle_repair_station`. Sure!